customer-config/checkout-config/che.tender.v1-preview.json

```diff
@@ -0,0 +1,174 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.tender.v1-preview.json",
+  "title": "Tender API",
+  "description": "The API allows you to define what tenders should be available in a given store\nand their respective behaviour, such as currency id, description,\ntender restriction rules for for specific item category etc.",
+  "type": "array",
+  "items": {
+    "$ref": "#/$defs/Tender"
+  },
+  "$defs": {
+    "Tender": {
+      "description": "Tender includes all the forms of payment that are accepted by the retail store\nin settling sales and other transactions.\n",
+      "type": "object",
+      "required": [
+        "id",
+        "currencyId",
+        "businessUnitGroupId"
+      ],
+      "properties": {
+        "businessUnitGroupId": {
+          "description": "Identifier for the business unit that this tender applies to.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 50,
+          "$comment": "store-1"
+        },
+        "id": {
+          "description": "A code which uniquely identifies the tender.\nIt is also the ID referenced to identify the tender in the POSLog\n(i.e. the transaction data from POS).\n",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 4,
+          "$comment": "cash"
+        },
+        "currencyId": {
+          "description": "A identifier for the currency. Usually the three letter ISO 4217 code.",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 3,
+          "$comment": "SEK"
+        },
+        "classCode": {
+          "description": "The group of tenders to which this tender type belongs. Can be\nCash, Check, CreditDebit, ForeignCurrency, GiftCertificate, OrganizationCharge,\\\nElectronicGiftCard, LoyaltyRedemption, Voucher, Coupon, FoodStamp, HouseAccount,\\\nWICCheck, ManufacturerCoupon\n",
+          "type": "string",
+          "minLength": 1,
+          "maxLength": 20,
+          "$comment": "Cash"
+        },
+        "name": {
+          "description": "A name for this tender.",
+          "type": "string",
+          "maxLength": 40,
+          "$comment": "PromotionCard"
+        },
+        "description": {
+          "description": "A description for this tender.",
+          "type": "string",
+          "maxLength": 255,
+          "$comment": "A promotion card"
+        },
+        "noAutoFinalization": {
```

Comment:
Wouldn't it be better to have the name `autoFinalization` and have the default value to true?

Reply:
Maybe have this as an additionalProperty instead. `Pos.autoFinalization`